PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# Pull Request Template
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
What is the reasoning behind this?

Reply:
not that much reasoning, didn't pay attention when going over a template i copied